lib/concurrent_guest_installations.pm

```diff
@@ -265,6 +265,21 @@ sub save_guest_installations_assets {
     return $self;
 }
 
+#Remove guests having unsuccessful installations according to setting KEEP_NORMAL_GUEST.
+sub clean_up_guests {
+    my $self = shift;
+
+    $self->reveal_myself;
+    if (get_var('KEEP_NORMAL_GUEST')) {
```

Comment:
Would you make it default to keeping only normal guest? This will let all testsuites stay with preferred behavior without setting new vars.

Reply:
Already added in this [merge request](https://gitlab.suse.de/qa-testsuites/openqa-job-group-yaml/-/merge_requests/298).

Reply:
I was suggesting more to skip setting this by updating a little bit code. You are adding it anywhere . Same effect, but different way... Not big deal.

Reply:
`get_var('KEEP_NORMAL_GUEST', '')` can avoid some "uninitialized ..." warnings, possibly :thinking: